src/plugins/console/server/routes/api/console/proxy/create_handler.ts

```diff
@@ -15,8 +15,7 @@ import { KibanaRequest, RequestHandler } from '@kbn/core/server';
 
 // TODO: find a better way to get information from the request like remoteAddress and remotePort
 // for forwarding.
-// eslint-disable-next-line @kbn/eslint/no-restricted-paths
-import { ensureRawRequest } from '@kbn/core/server/http/router';
+import { ensureRawRequest } from '@kbn/core-http-router-server-internal';
```

Comment:
I added a point to check for usages of this and decide if we want to expose it from the public package in my follow-up list.